app/assets/javascripts/admin/enterprises/controllers/enterprise_controller.js.coffee

```diff
@@ -1,13 +1,12 @@
 angular.module("admin.enterprises")
-  .controller "enterpriseCtrl", ($scope, $window, NavigationCheck, enterprise, EnterprisePaymentMethods, EnterpriseShippingMethods, SideMenu, StatusMessage) ->
+  .controller "enterpriseCtrl", ($scope, $window, NavigationCheck, enterprise, enterpriseRoles, EnterprisePaymentMethods, EnterpriseShippingMethods, SideMenu, StatusMessage) ->
     $scope.Enterprise = enterprise
+    $scope.EnterpriseRoles = enterpriseRoles
```

Comment:
I cannot for the life of me see where this is being used in the view. `app/views/admin/enterprises/form/_users.html.haml` seems to be using `Enterprise.users`?